docs/antora/modules/ROOT/pages/Library_Dependencies.adoc

```diff
@@ -150,11 +150,11 @@ def compileIvyDeps = Agg(
 )
 ----
 
-It is also possible to use higher version of the same library dependencies already defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+It is also possible to use a higher version of the same library dependencies already defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
 
 == Detecting transitive dependencies
 
-To render a tree of dependencies (transitive included) and possible upgrades for a given module, you can run `mill myModule.ivyDepsTree`. Here is how the start of `./mill __.ivyDepsTree` looks like in the `mill` project itself:
+To render a tree of dependencies (transitive included) you can run `mill myModule.ivyDepsTree`. Here is how the start of `./mill __.ivyDepsTree` looks like in the `mill` project itself:
```

Comment:
I changed this slightly because I don't think people will interpret the following correctly:

> and possible upgrades for a given module

When I read this I automatically think,  O mill will show me what needs to be updated? But that's not really what we're trying to say here I believe. While the tree may show you conflicting versions, I think we should leave this small part out to not confuse readers.